docker/prodlike/config/signer-config.toml

```diff
@@ -0,0 +1,261 @@
+# DISCLAIMER! READ!
+# This configuration file is an example of how it will likely look in production with some
+# values filled in with example placeholders like `your-private-key` and `your-bitcoin-node`.
+# The real production configuration will rely on some hardcoded values that can only be
+# known after initial seed node deployments. This file is not meant to be used as is, but one will
+# be created to be used later, and this documentation will be updated to reflect those changes.
+
+# TODO(715): Provide sane/safe configuration defaults. Re-review all of them!
+# TODO(429): Add documentation for all configuration parameters.
+
+# !! ==============================================================================
+# !! Blocklist Client Configuration
+# !! ==============================================================================
+[blocklist_client]
+
+# You may specify a blocklist client url. If one is not specified, then
+# deposit or withdrawal requests are always accepted.
+#
+# Format: "http(s)://<host>:<port>"
+# Default: <none>
+# Required: false
+# Environment: SIGNER_BLOCKLIST_CLIENT__ENDPOINT
+#
+# Defined in the provided docker compose, do not uncomment unless you know
+# what you're doing.
+# -------------------------------------------------------------------------
+# endpoint = "http://127.0.0.1:8080"
+
+# !! ==============================================================================
+# !! Emily API Configuration
+# !! ==============================================================================
+[emily]
+# The URI(s) of the Emily API server to connect to.
+#
+# You may specify multiple Emily API servers if you have them. They will be
+# tried round-robin until one succeeds.
+#
+# Format: ["http(s)://<host>:<port>", ..]
+# Default: <none>
+# Required: true
+# Environment: SIGNER_EMILY__ENDPOINTS
+endpoints = [
+    "https://emily-sbtc.com"
+]
```

Comment:
This needs to be changed to consume the configuration changes made in this ticket:
- #861